addons/collapse-footer/userscript.js

```diff
@@ -29,16 +35,19 @@ export default async function ({ addon, console }) {
 
   let collapseTimeout;
 
-  footer.addEventListener("mouseover", () => {
+  footer.addEventListener(addon.settings.get("mode") === "click" ? "click" : "mouseover", () => {
     footer.classList.add("transition", "expanded");
     if (collapseTimeout) {
       clearTimeout(collapseTimeout);
     }
   });
 
-  footer.addEventListener("mouseout", () => {
-    collapseTimeout = setTimeout(() => {
-      footer.classList.remove("expanded");
-    }, 200);
-  });
+  if (addon.settings.get("mode") === "click") {
+    document.addEventListener("click", (event) => {
```

Comment:
This could be changed to a `mousedown` event to collapse the footer as soon as you press your mouse button somewhere outside the footer, which might be more responsive if, for example, you long click or drag. It also triggers on right click and middle click.
```suggestion
    document.addEventListener("mousedown", (event) => {
```

_Expanding_ on `click` seems fine, though.